lib/util.js

```diff
@@ -159,9 +159,9 @@ function stylizeNoColor(str, styleType) {
 
 
 function arrayToHash(array) {
-  var hash = {};
+  var hash = Object.create(null);;
 
-  array.forEach(function(val, idx) {
+  array.forEach(function(val) {
```

Comment:
Ironically @JungMinu has already started some of this :) https://github.com/nodejs/node/pull/3622

Hopefully we'll be able to finish conversion of other applicable cases, but it's not high priority. Don't think the CTC would feel the need to make a stance "official" but if the rest of the code is following some convention then we can point to that as an example going forward.